tests/test_models/nn/test_deepar.py

```diff
@@ -2,6 +2,7 @@
 
 import pandas as pd
 import pytest
+from lightning_fabric.utilities.seed import seed_everything
```

Comment:
It seems like we don't have that package in pyproject.toml

Reply:
As I understand, it is a part of pytorch_lightning package: [source](https://lightning.ai/docs/fabric/stable/fundamentals/installation.html). I first tried to use [pytorch_lightning.utilities.seed](https://pytorch-lightning.readthedocs.io/en/1.6.5/api/pytorch_lightning.utilities.seed.html), but it is deprecated in favor of `lightning_fabric.utilities.seed`.